docs/contribute-docs.md

```diff
@@ -30,26 +30,47 @@ To build the MyST guide documentation:
 
 This will build the documentation locally so that you can preview what your changes will look like.
 
-## How this relates to the documentation at mystmd.org
+## How to update the content at mystmd.org
+
+When you change the content here or in most other MyST repositories, it will _not_ automatically update mystmd.org. You must take one extra step, described below.
+
+The content at mystmd.org is automatically updated with [a GitHub Action to deploy to our Vercel project](https://github.com/jupyter-book/mystmd.org/blob/main/.github/workflows/deploy.yml). When that action is run, it will pull the latest docs content from several MyST repositories, and push them to our Vercel deployment.
```

Comment:
@rowanc1 can you remind me if this is intentional that we don't deploy automatically? I could set up a web hook to trigger the deploy workflow.

Reply:
@agoose77 could we make that a follow-up issue? Or do you want to block this PR on that question?

Reply:
This is _not_ a blocker. I'm just looking for clarity on _why_ we have this policy. We should prioritise getting these docs merged, and revisit if needs be.

Reply:
I recommend:

- Creating an issue called `Make mystmd.org automatically update any time changes are merged into content repositories`
- Our first action in that issue should be to decide if we want to do it at all, or if something precludes us doing it.
- If we do want to do it, then the next step is to implement whatever webhooks you're talking about.
- If we don't want to do it, we write down a rationale for why not in that issue, and we close it as a record of the conversation+decision so that we don't have this uncertainty in the future